x-pack/docs/en/security/fips-140-compliance.asciidoc

```diff
@@ -147,9 +146,25 @@ for the file realm and the <<security-api-put-user,create users>> and
 <<security-api-change-password,change password>> APIs for the native and reserved realms.
 Other types of realms are not affected and do not require any changes.
 
+[discrete]
+==== Cached Password Hashing
```

Comment:
I went with "Cached" instead of "In-Memory" because the section is specifically about cached passwords and cache settings, and we have no cases of in-memory password hashing that _isn't_ cache hashing.